Dockerfile

```diff
@@ -4,7 +4,8 @@ MAINTAINER siddontang
 
 COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
-RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
+RUN apk add --no-cache mariadb-client && \
```

Comment:
can you use another RUN for apk especially?

Reply:
Yes.
I thought the mysqldump is required by go-mysql-elasticsearch, so made them into one layer.

Reply:
Mostly, installing pkg should be separated from others. It is cache friendly for docker.